package.json

```diff
@@ -109,6 +111,8 @@
     "react-native": ">=0.63.4",
     "react-native-gesture-handler": ">=1.9.0",
     "react-native-reanimated": ">=1.13.2",
+    "react-native-svg": "^12.1.0",
+    "react-native-svg-transformer": "^0.14.3",
```

Comment:
We should reconsider adding optional dependencies in our peer deps. 
If the dependency is optional we maybe don't want our users to get warnings about it (not until the use the relevant features). WDYT?

Reply:
I agree, removed (only the ones added in this PR)